### PR TITLE
Add dot exporting

### PR DESCRIPTION
**Pull request by Kaasiak**
Proposal to solving #27 

I added the possibility for the user to directly work with the graph object generated with PyDotPlus. `example.py` contains a a working example of how one can modify some edges / nodes after the graph is created and how it can be saved to any format file supported by Graphviz, such as .dot or .pdf.

### CODE REVIEW

File: src/cegpy/trees/event.py [hunk 1]
```diff
@@ -140,6 +140,9 @@ def __init__(self, dataframe, sampling_zero_paths=None,
         self.__construct_event_tree()
         logger.info('Initialisation complete!')
 
+        # creat the dot graph representation
```
Comment (g-walley) [inline src/cegpy/trees/event.py:143]: Typo: 
# Create*

File: src/cegpy/trees/event.py [hunk 2]
```diff
@@ -140,6 +140,9 @@ def __init__(self, dataframe, sampling_zero_paths=None,
...
+        self._dot_graph = None
```
Comment (g-walley) [inline src/cegpy/trees/event.py:144]: self._dot_graph appears unused. Do we need this?

File: src/cegpy/trees/event.py [hunk 3]
```diff
@@ -294,15 +301,15 @@ def create_figure(self, filename):
...
+        graph = self.dot_graph
         logger.info("--- writing " + filetype + " file ---")
         graph.write(str(filename), format=filetype)
 
         if get_ipython() is None:
             return None
         else:
             logger.info("--- Exporting graph to notebook ---")
-            return Image(graph.create_png())
+            return Image(self.graph.create_png())
```
Comment (g-walley) [inline src/cegpy/trees/event.py:312]: I don't think we need to add self here. 
graph.create_png() should still work.

**Review by g-walley — CHANGES_REQUESTED** [review]
Thanks for adding this, there are a few things I think need looking at before merging. Do you think they are justified?

File: src/cegpy/trees/event.py (hunk 2 above)
Comment (Kaasiak) [reply src/cegpy/trees/event.py:144]: You are right, this is redundant.

**Review by g-walley — APPROVED** [review]
Thanks, looks good

**Pull request merged**